app/src/main/java/org/mozilla/tv/firefox/channels/ChannelRepo.kt

```diff
@@ -63,58 +63,60 @@ class ChannelRepo(
                 pinnedTileRepo.removePinnedTile(tileData.url)
             }
             TileSource.POCKET -> throw NotImplementedError("pocket shouldn't be able to remove tiles")
-            TileSource.TV_GUIDE -> Unit // TODO in #2326
+            TileSource.TV_GUIDE -> {
+
+            }
         }
     }
 
     /**
      * Used to handle removing bundle tiles by adding to its [BundleType] blacklist in SharedPreferences
      */
     private fun addBundleTileToBlackList(type: BundleType, id: String) {
-        val blackList = loadBlackList(type).toMutableSet()
+        val blackList = loadBlackList(type).toMutableList()
```

Comment:
Hm, I think we were using a set so we didn't have to worry about duplicate values.  Why are we moving to a list here?

Reply:
Because blackListedIds require List type and all the other links thereafter:
i.e
`fun getNewsTiles(): Observable<List<ChannelTile>> `
`private val blacklistedNewsIds = BehaviorSubject.createDefault(emptyList<String>())`
...etc